docs/getting-started/apple/setup.md

```diff
@@ -1,4 +1,4 @@
-# fastlane guide
+# Getting started with fastlane for iOS
```

Comment:
I've been just saying "Apple" so far, since we support a number of different Apple platforms... I don't know how you feel about that!